### ISSUE TITLE
Captum Insights Widget wont render in Windows Anaconda Jupyter environment

### ISSUE THREAD

**Issue by MercyPrasanna** [pytorch/captum#769]
## 🐛 Bug

1. Insights widget fails to render in the jupyter notebook when I run - visualizer.render()

![image](https://user-images.githubusercontent.com/1892227/133914611-d6f01f2e-bbf0-4e4e-b1e4-1a758001c810.png)

2. When I run visualizer.serve(debug=True) and try to fetch the data - Fetch operation keeps running as shown below and the jupyter kernel dies.

![image](https://user-images.githubusercontent.com/1892227/133915201-79dddd0d-594b-410b-abf7-af5b16c89c53.png)

I checked if the data loader is returning the batches properly and it is working fine as seen below,
![image](https://user-images.githubusercontent.com/1892227/133915188-9242ce3b-5474-4c2c-80fa-5553fc10e590.png)

## To Reproduce

Run the below insights tutorial on Windows 10 anaconda jupyter environment
https://captum.ai/tutorials/CIFAR_TorchVision_Captum_Insights

## Expected behavior

1. Insights widget should be rendered in the jupyter notebook when I run visualizer.render()
2.  When I run visualizer.serve(debug=True) and try to fetch the data, it should succeed.

## Environment
Describe the environment used for Captum

 - Captum / PyTorch Version/Python Version
![image](https://user-images.githubusercontent.com/1892227/133915403-2d7144f3-a958-4b8c-8513-b3cf332069ea.png)

 - OS: 
 Windows
 
 - How you installed Captum / PyTorch (`conda`, `pip`, source):
 conda/pip

 - GPU models and configuration:
  running on cpu

## Additional context - Logs

python -m captum.insights.example
C:\Users\meranjit\AppData\Roaming\Python\Python37\site-packages\captum\insights\example.py:7: UserWarning: Deprecated. Please import from captum.insights.attr_vis.example instead.
  "Deprecated. Please import from captum.insights.attr_vis.example instead."
 * Debugger is active!

Fetch data and view Captum Insights at http://localhost:56754/

 * Debugger PIN: 227-680-127
 * Running on http://127.0.0.1:56754/ (Press CTRL+C to quit)
127.0.0.1 - - [19/Sep/2021 10:48:02] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [19/Sep/2021 10:48:02] "GET /static/js/2.c6c4604e.chunk.js HTTP/1.1" 200 -
127.0.0.1 - - [19/Sep/2021 10:48:02] "GET /static/css/main.fac91593.chunk.css HTTP/1.1" 200 -
127.0.0.1 - - [19/Sep/2021 10:48:02] "GET /static/js/main.9a5093fc.chunk.js HTTP/1.1" 200 -
127.0.0.1 - - [19/Sep/2021 10:48:02] "GET /init HTTP/1.1" 200 -
127.0.0.1 - - [19/Sep/2021 10:48:02] "GET /init HTTP/1.1" 200 -
127.0.0.1 - - [19/Sep/2021 10:48:02] "GET /favicon.ico HTTP/1.1" 404 -
Files already downloaded and verified
C:\Users\meranjit\AppData\Roaming\Python\Python37\site-packages\captum\insights\example.py:7: UserWarning: Deprecated. Please import from captum.insights.attr_vis.example instead.
  "Deprecated. Please import from captum.insights.attr_vis.example instead."
 * Debugger is active!

Fetch data and view Captum Insights at http://localhost:52339/

 * Debugger PIN: 227-680-127
 * Running on http://127.0.0.1:52339/ (Press CTRL+C to quit)
C:\Users\meranjit\AppData\Roaming\Python\Python37\site-packages\captum\insights\example.py:7: UserWarning: Deprecated. Please import from captum.insights.attr_vis.example instead.
  "Deprecated. Please import from captum.insights.attr_vis.example instead."

Fetch data and view Captum Insights at http://localhost:52340/

 * Debugger is active!
C:\Users\meranjit\AppData\Roaming\Python\Python37\site-packages\captum\attr\_utils\batching.py:152: UserWarning: It looks like that the attribution for a gradient-based method is
            computed in a `torch.no_grad` block or perhaps the inputs have no
            requires_grad.
  requires_grad."""
 * Debugger PIN: 227-680-127
OMP: Error #15: Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.
OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
 * Running on http://127.0.0.1:52340/ (Press CTRL+C to quit)

(vision-notebooks) C:\Users\meranjit>

**Comment by bilalsal** [comment]
Hi @MercyPrasanna ,

Thank you for reporting this issue! Could you please check your browser's console (e.g. right-click -> inspect -> Console in Chrome)? It would be helpful to know if there are any error messages there.

Thanks!

**Comment by MercyPrasanna** [comment]
I figured out the issue.  It was due to pyplot not rendering properly on jupyter. After applying the fix mentioned here https://github.com/matplotlib/matplotlib/issues/21513, insights rendered fine.
